### ISSUE TITLE
Add country name to `/visits` responses

### ISSUE THREAD

**Issue by billglover** [billglover/travel-log#20]
Queries to the following endpoints return `country_id` and not the name of the country. It is possible to look up the country name using `/countries/{.id}` but most uses of the `/visits` API will be to display visits by a particular user and will require the country name.

* `/visits`
* `/visits/{.id}`

Current visit object:
```json
{
  "id": 1,
  "user_id": 1,
  "country_id": 1
}
```

Desired visit object:
```json
{
  "id": 1,
  "user": {
    "id": 1
  },
  "country": {
    "id": 1,
    "name": "Canada"
  }
}
```

Modifying the `visit` object to include nested objects for both `country` and `user` will give us the flexibility to include further properties for these entities in future. This specific issue requires the `country` object to include both the `id` and the `name` property.

#### Hints:

* The country name is not stored in the `visits` table and so a join will be required
* You will get a flat object back from your database query. You will need to decide where to convert it to a nested object; model or controller.
* Existing tests might need re-work as we are changing the format of API responses.
* New tests will be required to check that calls to `/visits` include the country name.

**Comment by berryjen** [comment]
I will be on this

**Comment by billglover** [comment]
Awesome, I've assigned it to you.

**Comment by billglover** [comment]
Thanks for the fix in #23. This has now been merged and deployed. You can see the changes live here: https://travel.glvr.io/visits
